From: Behdad Esfahbod <behdad@behdad.org>
Date: Wed, 17 Oct 2018 15:08:03 -0700
Subject: [emoji Fix keycap sequence

Updated in Chromium.

(cherry picked from commit d4da231dfa53924ae074dece0cca93d49f39d30a)
---
 pango/emoji_presentation_scanner.rl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pango/emoji_presentation_scanner.rl b/pango/emoji_presentation_scanner.rl
index 5eea495..c13ae27 100644
--- a/pango/emoji_presentation_scanner.rl
+++ b/pango/emoji_presentation_scanner.rl
@@ -51,7 +51,7 @@ emoji_flag_sequence = REGIONAL_INDICATOR REGIONAL_INDICATOR;
 # https://www.unicode.org/reports/tr51/#def_emoji_tag_sequence
 emoji_tag_sequence = TAG_BASE TAG_SEQUENCE+ TAG_TERM;
 
-emoji_keycap_sequence = KEYCAP_BASE COMBINING_ENCLOSING_KEYCAP;
+emoji_keycap_sequence = KEYCAP_BASE VS16 COMBINING_ENCLOSING_KEYCAP;
 
 emoji_zwj_element =  emoji_presentation_sequence | emoji_modifier_sequence | any_emoji;
 
